.travis.yml

```diff
@@ -15,9 +15,12 @@ matrix:
     sudo: false
     addons:
       apt:
+        sources:
+        - kalakris-cmake
```

Comment:
This was necessary to add to get a newer version of cmake. The provided one in the version of Ubuntu that Travis is providing is quite out of date.